app/views/categories/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_with model: @category, method: :post, url: categories_path, scope: :new_category, local:true do |form| %>
+  <h3 class="display-3">Create Category</h3>
+  <hr>
+  <div class="mb-3">
+    <%= form.label :name,class:"form-label"%>
+      <%= form.text_field :name,class:"form-control", :required=> true %>
+  </div>
+  <div class="mb-3">
+    <%= form.label :icon,class:"form-label"%>
+      <%= form.text_field :icon,class:"form-control", :required=> true %>
+  </div>
```

Comment:
- [ ] [OPTIONAL] While the process of having to get the image link from a website will work perfectly fine, I would like to challenge you to try implementing the image upload functionality. This way your users can upload images directly from their local devices. [Here](https://medium.com/@anaharris/how-to-add-image-upload-functionality-to-your-rails-app-9f7fc3f3d042) is an amazing article for implementing this using [ActiveStorage](https://guides.rubyonrails.org/v6.0.0/active_storage_overview.html).